FusionIIIT/applications/visitor_hostel/models.py

```diff
@@ -125,6 +123,7 @@ class Bill(models.Model):
     meal_bill = models.IntegerField(default=0)
     room_bill = models.IntegerField(default=0)
     payment_status = models.BooleanField(default=False)
+    bill_date = models.DateField(default="00000000", blank=True)

```

Comment:
Refer [https://stackoverflow.com/questions/2029295/django-datefield-default-options](url) and modify the default accordingly.

Reply:
Solved